testing/python/metal/test_metal_threadsync_simdgroup_store.py

```
@@ -0,0 +1,423 @@
+
+
+@tilelang.jit
+def _access_ptr_raw(x, out, transpose, stride, threads=64):
+    """Cross-warp RAW through the T.access_ptr pointer form: warp 0 stores
+    the tile at sh[1, 0] (rows 1..8), warp 1 loads the tile at sh[8, 0]
+    (rows 8..15). The footprints overlap at row 8 while the base elements
+    are disjoint, so the pre-fix single-element disjointness proof skipped
+    the RAW barrier (Codex HIGH #1)."""
+    x: T.Tensor((8, 16), "float32")
+    out: T.Tensor((8, 16), "float32")
+    with T.Kernel(1, threads=threads) as (bx,):
```

Ruff (0.16.1)

[warning] Unpacked variable `bx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)